.github/workflows/site-deploy.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         with:
           personal_token: ${{ secrets.XTABLE_SITE_DEPLOY }}
           external_repository: apache/incubator-xtable-site
-          keep_files: false
+          keep_files: true
```

Comment:
Why is this changing?

Reply:
Checked few other apache project as well we don't need to keep this flag. 
https://github.com/peaceiris/actions-gh-pages?tab=readme-ov-file#%EF%B8%8F-keeping-existing-files-keep_files